core/templates/dev/head/pages/thanks.html

```diff
@@ -0,0 +1,53 @@
+{% extends "base.html" %}
+
+{% block subtitle %}
```

Comment:
I think this should be
"""
{% block maintitle %}
    Thanks - Oppia
{% endblock maintitle %}
"""
It's the title of the page.